src/webapp/components/data-submission/UploadRis.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ChangeEvent, useEffect, useRef, useState } from "react";
+import { Button } from "@material-ui/core";
+import styled from "styled-components";
+import i18n from "@eyeseetea/d2-ui-components/locales";
+import BackupIcon from "@material-ui/icons/Backup";
+import CloseIcon from "@material-ui/icons/Close";
+
+interface UploadRisProps {
+    handleValidate: (val: boolean) => void;
+}
+export const UploadRis: React.FC<UploadRisProps> = ({ handleValidate }) => {
+    const [fileList, setFileList] = useState<FileList | null>(null);
+
+    useEffect(() => {
+        if (fileList && fileList?.length) {
+            handleValidate(true);
+        } else {
+            handleValidate(false);
+        }
+    }, [fileList, handleValidate]);
+
+    const inputRef = useRef<HTMLInputElement>(null);
```

Comment:
Having to work directly with input[type=file] is a bit clunky in React. Just as a note, to consider with the project manager, there are libraries like https://react-dropzone.js.org/ (we use it in other projects).

Reply:
follow-up on: https://app.clickup.com/t/860pp77xz